### DISCUSSION TITLE
problem with installation

### DISCUSSION THREAD

**Discussion by akash-raneso** [luyadev/luya#2185] [General & Uncategorized]
https://github.com/luyadev/luya
I want to install this application but the installation guide page not working , if I install from the  installation guide of luya.io then I see this is another application that is https://github.com/luyadev/luya-kickstarter.

**Comment by nadar** [comment]
Hi @akash-raneso. i fixed those broken links thanks. yes the installation guide is here https://luya.io/guide/installation/ which uses the luya kickstarter you mentioned.

**Comment by akash-raneso** [comment]
I have installed through this https://luya.io/guide/installation/  but file structure are not same as give in application structure in luya.io , there are missing modules file , how can I resolve these?.
This is application structure layout in luya.io
├── assets
├── blockgroups
├── blocks
├── configs
├── filters
├── messages
├── migrations
├── models
├── modules
│   ├── <APP-MODULE>
│   │   ├── assets
│   │   └── controllers
│   └── <APP-ADMIN-MODULE>
│       ├── assets
│       └── controllers
├── public_html
│   ├── storage
│   └── css
├── resources
├── runtime
├── themes
└── views
    ├── <APP-MODULE>
    │   └── default
    ├── cmslayouts
    └── layouts 

And this is my application structure
├── assets
├── configs
├── messages
├── public_html
│   ├── storage
│   └── css
├── resources
├── runtime
└── views
    ├── cmslayouts
    └── layouts
